### PR TITLE
UCP/WIREUP: Minor code cleanups in address code

### PR DESCRIPTION
**Pull request by dmitrygx**
## What

1. Rename `UCT_ADDRESS_FLAG_ATOMIC*` to `UCP_ADDRESS_FLAG_ATOMIC*`
2. Introduce pack/unpack flags functions
3. Remove trailing empty line at the end of the `ucp/wireup/address.c` file

## Why ?

1. This is UCP code, they should have UCP prefix
2. To simplify code and will use them later in #4962 PR to pack/unpack event flags in the same way
3. Should have only one empty line at the end of the source file

## How ?

1. Rename macros and fix all places where they are used
2. Move pack and unpack code to `ucp_address_pack_flags`/`ucp_address_unpack_flags` and use them instead
3. Remove an excessive line

### CODE REVIEW

**Comment by dmitrygx** [comment]
bot:pipe:retest

**Comment by dmitrygx** [comment]
@evgeny-leksikov @brminich @hoopoepg could you review pls?
this was a part of #4962

File: src/ucp/wireup/address.c [hunk 1]
```diff
@@ -312,16 +312,57 @@ static void ucp_address_memcheck(ucp_context_h context, void *ptr, size_t size,
     }
 }
 
+static uint32_t ucp_address_pack_flags(uint64_t input_flags,
+                                       uint64_t cap_flags,
```
Comment (hoopoepg) [inline src/ucp/wireup/address.c:316]: minor: may be name it cap_mask?

File: src/ucp/wireup/address.c [hunk 2]
```diff
@@ -312,16 +312,57 @@ static void ucp_address_memcheck(ucp_context_h context, void *ptr, size_t size,
...
+                                       uint8_t output_start_bit)
+{
+    uint32_t result_flags = 0;
+    uint32_t packed_flag;
+    uint8_t cap_index;
+
+    ucs_assert(output_start_bit < 32);
```
Comment (hoopoepg) [inline src/ucp/wireup/address.c:323]: ucs_assert((ucs_popcount(cap_flags) + output_start_bit) < 32)

File: src/ucp/wireup/address.c [hunk 3]
```diff
@@ -312,16 +312,57 @@ static void ucp_address_memcheck(ucp_context_h context, void *ptr, size_t size,
...
+
+static uint64_t ucp_address_unpack_flags(uint32_t input_flags,
+                                         uint64_t cap_flags,
+                                         uint8_t input_start_bit)
+{
+    uint64_t result_flags = 0;
+    uint32_t packed_flag;
+    uint8_t cap_index;
+
+    ucs_assert(input_start_bit < 32);
```
Comment (hoopoepg) [inline src/ucp/wireup/address.c:345]: ucs_assert((ucs_popcount(cap_flags) + output_start_bit) < 32)

File: src/ucp/wireup/address.c (hunk 1 above)
Comment (dmitrygx) [reply src/ucp/wireup/address.c:316]: done

File: src/ucp/wireup/address.c (hunk 2 above)
Comment (dmitrygx) [reply src/ucp/wireup/address.c:323]: yes, makes sense to have this check

File: src/ucp/wireup/address.c (hunk 3 above)
Comment (dmitrygx) [reply src/ucp/wireup/address.c:345]: yes, makes sense to have this check

**Comment by dmitrygx** [comment]
@hoopoepg your comments were applied. could you check plz?

**Comment by dmitrygx** [comment]
@evgeny-leksikov @brminich

**Comment by dmitrygx** [comment]
@yosefe could you review pls?

**Pull request merged**